### DISCUSSION TITLE
Windows 11 KB5022913 preview update may break ExplorerPatcher

### DISCUSSION THREAD

**Discussion by tallman69** [Open-Shell/Open-Shell-Menu#1340] [General]
Since some of us who use Open-Shell also use ExplorerPatcher, and since the public forum for ExplorerPatcher is closed, I thought I would give a heads up here to those of you who use ExplorerPatcher.  According to this article on Bleeping Computer, the newly released Windows 11 KB502913 preview update may cause Windows 11 to not boot if you have ExplorerPatcher installed on your machine:

https://www.bleepingcomputer.com/news/microsoft/windows-11-kb5022913-causes-boot-issues-if-using-ui-customization-apps/

I have not tested the update on either of my Windows 11 machines.  Hopefully valinet will come out with a fix for ExplorerPatcher before Microsoft forces installation of KB5022913 in March.

**Comment by JYewdall** [comment]
I am seeing this too late!
Today installed kb5022906 (2023-2 Cumulative Update) on Windows 11 and rebooted to a flashing blank desktop and empty taskbar. It required a reinstall. After hours rebuilding my pc I installed explorerpatcher and bang, instantly bad again.

**Reply by tallman69** [reply]
Sorry to hear that!  I was not familiar with KB5022906 because this update is for Windows 10 not Windows 11.  I had not read that KB5022906 did the same thing to Windows 10 that KB5022913 did to Windows 11.

Did you try to uninstall ExplorerPatcher in Safe Mode before resorting to a reinstall of Windows?  https://www.tenforums.com/tutorials/143263-how-install-uninstall-programs-safe-mode-windows-10-a.html

**Reply by JYewdall** [reply]
I think i have it wrong on the KB number.
On the reinstall i am not being offered 2023-2 cumulative update on my windows 11 machine. I assumed it was the same KB as the 2023-2 update on my other machine which is windows 10. 
I couldn'tuninstall. First time round I had a user that had not previously logged on so I could open a cmd session and use wmic to uninstall software but wasn't aware explorerpatcher was the cause. Second time round, knowing explorerpatcher was the cause I attempted to uninstall it but both my users had previously logged on. Explorer was repeatedly restarting every few seconds so the focus was being taken away when typing and it was impossible to type the command once I managed to launch a cmd session. For anyone faced with this problem they need a user that has not logged on previously. Then ctl-alt-del to launch task manager. Then run new task - cmd as administrator. i think the command should be
**wmic product where name="ExplorerPatcher" call uninstall**
i can't be certain about the product name as it is no longer installed  **wmic product get name** lists the products installed

**Reply by tallman69** [reply]
Do you have ExplorerPatcher on your Windows 10 machine?  I am wondering if the 2023-2 cumulative update preview for Windows10 is affected by ExplorerPatcher like it is on Windows 11.

Do you know if it is possible to boot into Safe Mode on Widows 11 after the 2023-2 update preview is installed with ExplorerPatcher on the machine?

**Comment by JYewdall** [comment]
I don't have explorerpatcher installed on windows 10. I think i needed it on windows 11 to get my taskbar vertical. You are supposed to be able to select safe mode from within the advanced startup. I didn't seem to be getting all the options though.
https://pureinfotech.com/access-advanced-startup-options-windows-11/
I had to use the power down during startup method to get into the advanced startup where I chose reset.

**Reply by tallman69** [reply]
Okay, thanks for the info!

**Comment by tallman69** [comment]
Valinet released the latest version of ExplorerPatcher (version 22621.1344.53.1) 5 hours ago that fixes the issue with the 2023-2 update preview.  That was fast!  

I installed the ExplorerPatcher update - so far so good.  I will wait until later in the month to install the KB5022913 update when Microsoft forces the install.  Hopefully any other bugs will get worked out of it by then.

**Comment by JYewdall** [comment]
I don't know whether I am that brave. Before I do install it I will create a new unused account to use if it goes wrong.

**Reply by tallman69** [reply]
I have windows 11 on two machines, one of which is a laptop that I use as a test machine.  I will install the update on it and see what happens.

**Reply by tallman69** [reply]
I installed the 2023-2 update preview on the laptop after updating ExplorerPatcher.  Everything went well without any problems.  The only thing I noticed was that the taskbar icons seemed to load slower than normal, but as far as I can tell everything is working fine.  Hope all goes well for you!
